hoomd/operations.py

```diff
@@ -7,12 +14,40 @@
 
 
 def _triggered_op_conversion(value):
+    """Handle converting _TriggeredOperation to a operation, trigger pair.
+
+    Necessary since in C++ operations do not own their trigger.
+    """
     return (value._cpp_obj, value.trigger)
 
 
 class Operations:
```

Comment:
That is not a bad idea. We could make it a `Collection`. We don't get any mixin methods for that though. On the `MutableSet` set idea, it is in a lot of ways like a set, but then using `+=` as a valid operator for the object may feel off for users since Python sets don't implement a `__iadd__`. Also, `-=` would change meaning and only work with other `Operations` objects if we are being true to the set concept.

Reply:
I vote to make it a `Collection` but not a set or anything more specific.